package.json

```diff
@@ -38,17 +38,18 @@
     "test": "jest --config jest.json"
   },
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-core": "^6.25.0",
+    "@babel/cli": "^7.0.0-beta.44",
+    "@babel/core": "^7.0.0-beta.44",
+    "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^7.2.3",
-    "babel-plugin-syntax-jsx": "^6.18.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.24.1",
+    "@babel/plugin-syntax-jsx": "^7.0.0-beta.44",
+    "@babel/plugin-transform-modules-commonjs": "^7.0.0-beta.44",
     "classnames": "^2.2.5",
     "eslint": "^4.1.1",
     "eslint-config-prettier": "^2.3.0",
     "glob": "^7.1.2",
     "husky": "^0.14.2",
-    "jest": "^20.0.4",
+    "jest": "^22.4.3",
```

Comment:
Can you please revert this `jest` upgrade? I think the new version introduces some formatting changes to the snapshot file. I prefer this PR to just upgrade `babel` and any requirements for  it.

I'll upgrade `jest` with a follow-up commit.

Reply:
jest had a direct dependency to babel-core, witch they fixed in jest 21: https://github.com/facebook/jest/blob/master/CHANGELOG.md#jest-2100

> Change babel-core to peerDependency for compatibility with Babel 7

Reply:
I can modify to 21 if you prefer